src/main/java/com/cinema/model/CinemaHall.java

```diff
@@ -0,0 +1,45 @@
+package com.cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class CinemaHall {
```

Comment:
let's add `@Table` annotation here to ensure that our table has `proper_name`

Reply:
done